pkg/tools/spire/options.go

```diff
@@ -49,11 +53,33 @@ func WithAgentID(agentID string) Option {
 }
 
 // WithEntry - Option to add Entry to spire-server.  May be used multiple times.
-func WithEntry(spiffeID, selector string) Option {
+func WithEntry(spiffeID, selector, federatesWith string) Option {
```

Comment:
This could break all cmd apps. Please consider about adding a new option `WitthFederatesEntry` or something like that

Reply:
Done